next/frontend/utils/universalStorage.ts

```diff
@@ -0,0 +1,116 @@
+import { environment } from 'environment'
```

Comment:
please reference the "source" after which this was modelled. Normally I'd write "documentation" but in case it was more a case of opening amplify sources and looking in there, link to those sources instead.

Also reference any other sources that helped along the way and were relevant to this - github issues / comments/ blog posts...